terraform/monitoring/variables.tf

```diff
@@ -66,7 +66,7 @@ variable "alertmanager" {
   }
 }
 
-variable "google_client_id" {}	
+variable "google_client_id" {}
```

Comment:
remove '"' from variable names and keep it consistent (as we are touching this file)?

Reply:
Next time it is changed I will pickit up